src/chapters/jdbc-auth.xml

```diff
@@ -413,12 +509,29 @@ postgresql-password: <replaceable>some_password</replaceable></programlisting>
                         <row>
                             <entry><property>mysql-password</property></entry>
                             <entry><property>postgresql-password</property></entry>
+                            <entry><property>sqlserver-password</property></entry>
                             <entry>
                                 <para>The password Guacamole should provide when authenticating with
                                     the database. This is given as "some_password" in the examples
                                     given in this chapter.</para>
                             </entry>
                         </row>
+                        <row>
+                            <entry></entry>
+                            <entry></entry>
+                            <entry><property>sqlserver-driver</property></entry>
+                            <entry>
+                                <para>The SQL Server driver has an additional required parameter to
+                                configure the TDS compatibility of the driver.  This driver supports
+                                four different options for this command:
+                                - <property>microsoft2005</property> for the current Microsoft driver,
```

Comment:
The `<property>` tag represents a property, not a possible value of a property. In this case, you probably want `<constant>`. There are also semantic tags for various types of lists here, if you want to represent this as a list, as the dashes will not result in this formatting as you might expect; they'll all be inline.

Formatting them inline is probably fine, honestly, but if that's the intent, I'd say remove the dashes.

Reply:
Fixed.